fuel-tx/src/transaction/consensus_parameters/gas.rs

```diff
@@ -321,6 +321,7 @@ pub struct GasCostsValues {
     // Non-opcode costs
     pub contract_root: DependentCost,
     pub state_root: DependentCost,
+    pub new_storage_per_byte: Word,
```

Comment:
I think it is better to use `DependentCost` here because we need to charge for a new storage slot, plus, in the future(when we support dynamic storage values), charge per byte.

Reply:
I don't think it makes any sense to future-proof this for the dynamic storage value support. The whole API is going to change anyway.